docs/examples/compute/instance/remote-exec.tf

```diff
@@ -1,19 +1,19 @@
 resource "null_resource" "remote-exec" {
-    depends_on = ["oci_core_instance.TFInstance","oci_core_volume_attachment.TFBlock0Attach"]
+    depends_on = ["oci_core_instance.TFInstance","oci_core_volume_attachment.TFBlockAttach"]
+    count = "${var.NumInstances * var.NumVolumesPerInstance}"
     provisioner "remote-exec" {
       connection {
         agent = false
         timeout = "30m"
-        host = "${data.oci_core_vnic.InstanceVnic.public_ip_address}"
+        host = "${oci_core_instance.TFInstance.*.public_ip[count.index % var.NumInstances]}"
         user = "opc"
         private_key = "${var.ssh_private_key}"
     }
       inline = [
         "touch ~/IMadeAFile.Right.Here",
-        "sudo iscsiadm -m node -o new -T ${oci_core_volume_attachment.TFBlock0Attach.iqn} -p ${oci_core_volume_attachment.TFBlock0Attach.ipv4}:${oci_core_volume_attachment.TFBlock0Attach.port}",
-        "sudo iscsiadm -m node -o update -T ${oci_core_volume_attachment.TFBlock0Attach.iqn} -n node.startup -v automatic",
-        "echo sudo iscsiadm -m node -T ${oci_core_volume_attachment.TFBlock0Attach.iqn} -p ${oci_core_volume_attachment.TFBlock0Attach.ipv4}:${oci_core_volume_attachment.TFBlock0Attach.port} -l >> ~/.bashrc"
+        "sudo iscsiadm -m node -o new -T ${oci_core_volume_attachment.TFBlockAttach.*.iqn[count.index]} -p ${oci_core_volume_attachment.TFBlockAttach.*.ipv4[count.index]}:${oci_core_volume_attachment.TFBlockAttach.*.port[count.index]}",
```

Comment:
All of this would go away with the change to `paravirtualized` volume attachments.